publications/index.html

```diff
@@ -6,13 +6,39 @@ <h1>How to cite BioSchemas</h1>
 To cite BioSchemas please use the website URL, https://bioschemas.org and "Gray, A.J.G, Goble, C.A. and Jimenez, R., 2017. Bioschemas: From Potato Salad to Protein Annotation. In International Semantic Web Conference (Posters, Demos & Industry Tracks)."
```

Comment:
Should it not be Bioschemas? We should use the same spelling everywhere

Reply:
Fixed in commit 968afdb